docs/pipeline-components-and-applications/enrichment-components/enrich-kafka/index.md

```diff
@@ -31,6 +31,8 @@ Above assumes that you have following directory structure:
 2. Iglu Resolver [configuration JSON](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md)
 3. [configuration HOCON](/docs/pipeline-components-and-applications/enrichment-components/configuration-reference/index.md)
 
+It is possible to use environment variables in all of the above.
```

Comment:
But for enrichments and Iglu this is only available since 3.7.0 right? I would clarify that.

Reply:
Good point ! I updated